src/towncrier/newsfragments/482.feature.rst

```diff
@@ -0,0 +1,5 @@
+If no filename is given when doing ``towncrier`` create, interactively ask for the issue number and fragment type (and then launch an interactive editor for the fragment content).
+
+Now by default, when creating a fragment it will be appended with the ``filename`` option's extension (unless an extension is explicitly provided). For example, ``towncrier create 123.feature`` will create ``news/123.feature.rst``. This can be changed in configuration file by setting `add_extension = false`.
+
+A new line is now added by default to the end of the fragment contents. This can be reverted in the configuration file by setting `add_newline = false` or explicitly set with the ``create`` command line flags ``--eof-newline/--no-eof-newline``.
```

Comment:
I am ok with this new default value... but maybe it should be placed into a separate section ... as it looks like a backward incompatible change.

Reply:
As in a separate fragment? I'm not sure where else you mean this should go.

Reply:
Usualy backward incopatible changes are announced into `482.removal.rst`  This is the "Deprecation and removal" category.